provider/resources.go

```diff
@@ -250,5 +252,12 @@ func Provider() tfbridge.ProviderInfo {
 
 	prov.SetAutonaming(255, "-")
 
+	err := prov.ComputeDefaults(tfbridge.TokensSingleModule("github_", mainMod,
+		func(module, name string) (string, error) {
```

Comment:
If you don't mind tidying up a bit while we're a it, this looks suspiciously similar to makeResource and makeDataSource above which also look identical to each other. Perhaps reuse or delete those?

Reply:
Sure. I can reuse `makeResource`.